.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,31 +6,33 @@ labels: bug
 assignees: ''
 ---
 
-**Describe the bug**
+## Describe the bug
 A clear and concise description of what the bug is.
 
-**To Reproduce**
+## To Reproduce
 Steps to reproduce the behavior:
 
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
-**Expected behavior**
+## Expected behavior
 A clear and concise description of what you expected to happen.
 
-**Screenshots**
+## Screenshots
 If applicable, add screenshots to help explain your problem.
 
-**Version info:**
-_Run `lazygit --version` and paste the result here_
-_Run `git --version` and paste the result here_
+## Version info:
 
-**Additional context**
+* _Run `lazygit --version` and paste the result here_
+* _Run `git --version` and paste the result here_
+
+## Additional context
 Add any other context about the problem here.
 
-**Note:** please try updating to the latest version or [manually building](https://github.com/jesseduffield/lazygit/#manual) the latest `master` to see if the issue still occurs.
+> [!NOTE]
```

Comment:
This part is rendered like:

<img width="950" height="230" alt="Image" src="https://github.com/user-attachments/assets/80e10d08-f920-4040-86ee-122fad3b7b63" />